Dockerfile.dapper

```diff
@@ -1,21 +1,20 @@
 FROM quay.io/costoolkit/releases-green:luet-toolchain-0.21.2 as luet
 FROM quay.io/costoolkit/releases-green:luet-makeiso-toolchain-0.4.0 as makeiso
 
-FROM golang:1.18.3-buster
+FROM golang:1.20.10-bookworm
```

Comment:
Just curious, why do we use `registry.suse.com/bci/golang:1.20`?
(I know the original commit uses the above one, just curious.)

Reply:
The Dockerfile.dapper is used to run scripts in `scripts` folder. The released image Dockerfile is https://github.com/harvester/harvester/blob/master/package/Dockerfile. I think it's okay to use any golang images for the building process.

Reply:
Thanks for the explanation!

Reply:
Let's keep the debian base image for now and move to BCI in newer versions.